tests/integration/test_runner_manager_openstack.py

```diff
@@ -202,22 +205,30 @@ def workflow_is_status(workflow: Workflow, status: str) -> bool:
     return workflow.status == status
 
 
-async def wait_runner_amount(runner_manager: RunnerManager, num: int):
+async def wait_runner_amount(
+    runner_manager: RunnerManager, num: int, timeout: int = 600, check_interval: int = 60
+) -> None:
     """Wait until the runner manager has the number of runners.
 
     A TimeoutError will be thrown if runners amount is not correct after timeout.
 
     Args:
         runner_manager: The RunnerManager to check.
         num: Number of runner to check for.
+        timeout: The timeout in seconds.
+        check_interval: The interval to check in seconds.
     """
     runner_list = runner_manager.get_runners()
     assert isinstance(runner_list, tuple)
     if len(runner_list) == num:
         return
 
     # The openstack server can take sometime to fully clean up or create.
-    await wait_for(lambda: len(runner_manager.get_runners()) == num)
+    await wait_for(
+        lambda: len(runner_manager.get_runners()) == num,
+        timeout=timeout,
+        check_interval=check_interval,
+    )
```

Comment:
Increased the timeout by default as the tests were quite brittle. Also increased the check_interval as there is so much logged on each run.